src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.ListView.cs

```diff
@@ -773,29 +763,129 @@ private void TasView_MouseDown(object sender, MouseEventArgs e)
 			}
 		}
 
+		/// <summary>
+		/// Begins a batch of edits, for auto-restore purposes. Auto-restore will be delayed until EndBatchEdit is called.
+		/// </summary>
+		private void BeginBatchEdit()
+		{
+			_batchEditing = true;
+		}
+		/// <returns>Returns true if the input list was redrawn.</returns>
+		private bool EndBatchEdit()
+		{
+			_batchEditing = false;
+			if (_batchEditMinFrame != -1)
+			{
+				return FrameEdited(_batchEditMinFrame);

```

Comment:
I know I said you shouldn't worry about mismatched calls, but `_batchEditMinFrame = -1;` here could prevent some bugs.